css/gallery.css

```diff
@@ -0,0 +1,116 @@
+/*class for the section of gallery*/
+.info-gallery {
+    width: 635px;
+    height: 88px;
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: normal;
```

Comment:
Do we have to specify this kind of -default- property values when they are not being previously modified in other classes? I had something similar in one if my PRs.

Reply:
Same for the next for properties

Reply:
Yes, "normal" should not be set unless a paren element is modifying your element by inheritance.

Reply:
Got it, then I think these properties should be removed.